Website/src/Foundation/BranchPresets/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("BranchPresets")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("BranchPresets")]
+[assembly: AssemblyCopyright("Copyright ©  2016")]
```

Comment:
2021?

Reply:
Updated, but not sure if it is right... usually copyright stays the same since the code was written.